core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay

```diff
@@ -35,3 +35,5 @@ registry.ci.openshift.org/ocp-kni/4.9:cnf-tests quay.io/openshift-kni/cnf-tests:
 registry.ci.openshift.org/ocp-kni/4.9:dpdk quay.io/openshift-kni/dpdk:4.9 quay.io/openshift-kni/dpdk:4.9.0
 registry.ci.openshift.org/ocp-kni/4.10:cnf-tests quay.io/openshift-kni/cnf-tests:4.10 quay.io/openshift-kni/cnf-tests:4.10.0 quay.io/openshift-kni/cnf-tests:latest
 registry.ci.openshift.org/ocp-kni/4.10:dpdk quay.io/openshift-kni/dpdk:4.10 quay.io/openshift-kni/dpdk:4.10.0 quay.io/openshift-kni/dpdk:latest
+# cluster-group-upgrades-operator
+registry.ci.openshift.org/ocp-kni/cluster-group-upgrades-operator:cluster-group-upgrades-operator quay.io/openshift-kni/cluster-group-upgrades-operator:4.10-snapshot
```

Comment:
it should be tag:image as the others.
Also, I don't see a repo created under quay

Reply:
I see your point here, but looking at PAO, I noticed the format is different for the latest release:
registry.ci.openshift.org/ocp-kni/performance-addon-operator:performance-addon-operator quay.io/openshift-kni/performance-addon-operator:4.10-snapshot. Given the cluster-group-upgrades-operator is only supported starting with 4.10, I thought I could follow the same model.

Kindly let me know if you agree.

Will look into the repo being created under quay. Thank you!

Reply:
Please note that PAO is not anymore a good example re: best practice. What PAO did is not necessarily the right thing to do in late 2021.